Shell.pm: add iDRAC 4.00.00.00 support

alip opened the pull request:
<!-- Thanks for contributing to Rex! -->
<!-- For optimal workflow, please make sure you have read and understood the [Contributing guide](https://github.com/RexOps/Rex/blob/master/CONTRIBUTING.md). -->

<!-- TL; DR: -->
<!-- Make sure there's an issue where the proposed changes are already discussed. -->
<!-- Please open the pull request as a draft first, and wait for automated test results. -->
<!-- Feel free to work on the PR till tests pass, and the checklist below is complete, then mark it ready for review. -->

This PR is an attempt to fix #<!-- issue ID --> by <!-- briefly explaining your changes -->.

<!-- Ideally, ask for a specific expected course of action, like: -->
<!-- Please review and merge, or let me know how to improve it further. -->

## Checklist

- [ ] based on top of latest source code <!-- Make sure your changes are based on the latest version of the source code, rebase your branch if necessary. -->
- [ ] changelog entry included <!-- If the change is interesting for the users or developers, it should be mentioned in the changelog. -->
- [ ] tests pass on Travis CI <!-- Demonstrate the code is solid. Include new tests first, let them fail, then push the fix, allowing tests to pass. -->
- [ ] git history is clean <!-- Ideally two commits are needed: one for adding new tests that fail, and one that fixes them. -->
- [ ] git commit messages are [well-written](https://chris.beams.io/posts/git-commit/#seven-rules)

ferki reviewed (changes requested):
I believe this can be handled pretty much the same way as #1468:

- fix the copyright
- publish module on CPAN
- include the mapping upstream

alip commented:
Thank you for the feedback. I have updated this PR to only include the mapping and created a git repository to host the module [here](https://github.com/adjust/Rex-Interface-Shell-Idrac4) where I fixed the copyright and formatted the code with perltidy. @ferki, would you please be so kind to take a quick look at the aforementioned repository before I make a release to see if it looks good? Thank you.

ferki commented:
As briefly discussed via IRC earlier, I'm closing this one in favor of discussing a more complete solution via #1481.
